From 2c95a483c175459abe6528200896e98f57adec7c Mon Sep 17 00:00:00 2001
From: Manjeet Rulhania <mrulhania@google.com>
Date: Thu, 30 Jun 2022 22:20:08 +0000
Subject: [PATCH] Remove package name from SafetyNet logs

Bug: 213323615
Test: AppSecurityTests
Change-Id: I8f823487ca16861a35135cfc3383fa2ce8258017
Merged-In: I4b61d13256ce0bfb8fc9d21db52ee78ce2097f14
(cherry picked from commit dfeea39dbc61ea9884e7e017335cbdf4942fa181)
Merged-In: I8f823487ca16861a35135cfc3383fa2ce8258017
---
 .../content/pm/parsing/component/ParsedPermissionUtils.java    | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/core/java/android/content/pm/parsing/component/ParsedPermissionUtils.java b/core/java/android/content/pm/parsing/component/ParsedPermissionUtils.java
index 8f8cba4b0c9..9b465947bce 100644
--- a/core/java/android/content/pm/parsing/component/ParsedPermissionUtils.java
+++ b/core/java/android/content/pm/parsing/component/ParsedPermissionUtils.java
@@ -274,8 +274,7 @@ public class ParsedPermissionUtils {
                 final ParsedPermission perm = checkDuplicatePerm.get(name);
                 if (isMalformedDuplicate(parsedPermission, perm)) {
                     // Fix for b/213323615
-                    EventLog.writeEvent(0x534e4554, "213323615",
-                            "The package " + pkg.getPackageName() + " seems malicious");
+                    EventLog.writeEvent(0x534e4554, "213323615");
                     return true;
                 }
                 checkDuplicatePerm.put(name, parsedPermission);
-- 
2.37.1.359.gd136c6c3e2-goog

